.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5501
+}
```

Comment:
- [OPTIONAL]

- [ ] You can add this directory to the `.gitignore` file so that it is not been pushed to Github as it is not necessary to be present in your repository.